src/features/Policy/PolicyModal/index.js

```diff
@@ -28,14 +37,22 @@ const PolicyModal = ({ open, handleClose }) => {
             footer={null}
             onCancel={handleCancel}
         >
-            <Form form={form} onFinish={handleFinish}>
+            <Form
+                form={form}
+                initialValues={{
+                    content: selectedPolicy?.content || '',
```

Comment:
for updated use useeffect and updated value in useeffect 
antd allready provide updated method 
  form.setFieldsValue({}); like this

Reply:
resolved